app/forms/idp/create.tsx

```diff
@@ -62,6 +65,24 @@ export function CreateIdpSideModalForm() {
   })
 
   const form = useForm({ defaultValues })
+  const name = form.watch('name')
+
+  const acsUrlForm = useForm({ defaultValues: { generateUrl: true } })
```

Comment:
This works, but it would be simpler as a regular `useState` and a checkbox instead of a CheckboxField